Mage.Server/src/main/java/mage/server/graphql/GameGraphqlPublisher.java

```diff
@@ -0,0 +1,66 @@
+package mage.server.graphql;
+
+import mage.server.graphql.data.GamePublisher;
+import graphql.schema.DataFetcher;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.RuntimeWiring;
+import graphql.schema.idl.SchemaGenerator;
+import graphql.schema.idl.SchemaParser;
+import graphql.schema.idl.TypeDefinitionRegistry;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.Collections;
+import java.util.List;
+
+import static graphql.schema.idl.TypeRuntimeWiring.newTypeWiring;
+
+public class GameGraphqlPublisher {
+    private final static GamePublisher STOCK_TICKER_PUBLISHER = new GamePublisher();
+
+    private final GraphQLSchema graphQLSchema;
+
+    public GameGraphqlPublisher() {
+        graphQLSchema = buildSchema();
+    }
+
+    private GraphQLSchema buildSchema() {
+        //
+        // reads a file that provides the schema types
+        //
+        Reader streamReader = loadSchemaFile("stocks.graphqls");
+        TypeDefinitionRegistry typeRegistry = new SchemaParser().parse(streamReader);
+
+        RuntimeWiring wiring = RuntimeWiring.newRuntimeWiring()
+                .type(newTypeWiring("Subscription")
+                        .dataFetcher("stockQuotes", stockQuotesSubscriptionFetcher())
+                )
+                .build();
+
+        return new SchemaGenerator().makeExecutableSchema(typeRegistry, wiring);
+    }
+
+    private DataFetcher stockQuotesSubscriptionFetcher() {
```

Comment:
Is this a hold over from a copy paste?

Reply:
Yes, working on fleshing that out now. It was originally a live stock quoting app. Just using it as a starting point.